src/pages/resultsView/ResultsViewPage.tsx

```diff
@@ -132,9 +132,21 @@ export default class ResultsViewPage extends React.Component<IResultsViewPagePro
 
         // hide mutex tab
         $(document).ready(()=>{
-            if (genes((window as any).serverVars.theQuery).length <= 1) {
+            if (!(window as any).serverVars.theQuery.trim().length || genes((window as any).serverVars.theQuery).length <= 1) {
```

Comment:
@adamabeshouse lets rename this function "genes" to something more descriptive.   it seems to parse query and return genes symbols, so something to that effect.